lib/galaxy/util/__init__.py

```diff
@@ -2022,3 +2024,13 @@ def to_content_disposition(target: str) -> str:
     sanitized_filename = "".join(c in FILENAME_VALID_CHARS and c or "_" for c in filename)[0:character_limit] + ext
     utf8_encoded_filename = quote(re.sub(r'[\/\\\?%*:|"<>]', "_", filename), safe="")[0:character_limit] + ext
     return f"attachment; filename=\"{sanitized_filename}\"; filename*=UTF-8''{utf8_encoded_filename}"
+
+
+def validate_doi(doi: str) -> bool:
+    if len(doi) > DOI_MAX_LENGTH:
+        return False
+    prefix = "https://doi.org/|doi.org/|doi:"
```

Comment:
I was looking into reusing this but we are forcing the prefix here which is not part of [the spec](https://www.doi.org/resources/DOICoreSpecificationv1.pdf) -- is there a reason I am missing?

Reply:
I've found that it may be recommended to use a URL prefix for DOI presention (I checked multiple sources). So while it is not part of the formal spec, it helps detect DOIs in the wild.  

Here're some of the sources I used:
https://support.datacite.org/docs/doi-basics#proxy
https://support.datacite.org/docs/datacite-doi-display-guidelines (linked from the prev source)
https://www.crossref.org/display-guidelines/
https://www.doi.org/doi-handbook/HTML/presentation-formats-of-a-doi-.html

Reply:
We are not in the wild though, we are in a field clearly marked DOI.

<img width="289" alt="Screenshot 2025-05-23 at 15 03 35" src="https://github.com/user-attachments/assets/e01833c3-5d54-4312-87c9-a8fc6c659ec1" />